admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -511,6 +511,25 @@ a look at the logs for further details."
 When the configuration test reports success, save your settings and check if the 
 users and groups are fetched correctly on the Users page.
 
+Syncing Users
+-------------
+
+You need to sync all LDAP users to your ownCloud database after they've been created, using :ref:`the occ command <syncing_user_accounts_label>`.
+Versions of ownCloud before 10.0 imported all users when the user page loaded, but this is no longer the case. 
+
+In versions of ownCloud from 10.0 onward, un-synced users *can* log in, **but**:
+
+#. They won't be listed in the user search.
```

Comment:
This should not be the case since this user should be correctly displayed in the users search and it should be possible to share with him once he first logged in.

Reply:
Oh ok. That's not how I understood it from what @cdamken told me.

Reply:
@pako81, I've updated that section. Can you have another look, please?

Reply:
@settermjd looks good to me, thanks